src/events/channel/ChannelEvent.js

```diff
@@ -0,0 +1,51 @@
+const BaseEvent = require("../../utils/structures/BaseEvent");
+
+module.exports = class ChannelEvent extends BaseEvent {
+  constructor() {
+    super("voiceStateUpdate");
+    this.mapVoiceToText = new Map([
+      ["818439901098934292", "819264453097947176"],
+    ]); //[voice channel, hidden text channel]
```

Comment:
Could an object be used instead of a map structure? If so, perhaps we could just store the voice channels and text channels in a json file. Wdyt?

Reply:
Already applied changes